### PR TITLE
Java: Add change note for LDAP injection query.

### PR DESCRIPTION
**Pull request by aschackmull**
Add a change note for the query added in https://github.com/Semmle/ql/pull/2651.

### CODE REVIEW

File: change-notes/1.24/analysis-java.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ The following changes in version 1.24 affect Java analysis in all applications.
 |-----------------------------|-----------|--------------------------------------------------------------------|
 | Disabled Spring CSRF protection (`java/spring-disabled-csrf-protection`) | security, external/cwe/cwe-352 | Finds disabled Cross-Site Request Forgery (CSRF) protection in Spring. |
 | Failure to use HTTPS or SFTP URL in Maven artifact upload/download (`java/maven/non-https-url`) | security, external/cwe/cwe-300, external/cwe/cwe-319, external/cwe/cwe-494, external/cwe/cwe-829 | Finds use of insecure protocols during Maven dependency resolution. Results are shown on LGTM by default. |
+| LDAP query built from user-controlled sources (`java/ldap-injection`) | security, external/cwe/cwe-090 | Finds LDAP queries vulnerable to injection of unsanitized user-controlled input. |
```
Comment (felicitymay) [inline change-notes/1.24/analysis-java.md]: Will this be run on LGTM? If so, will the results be shown or hidden, by default?

Comment (aschackmull) [reply change-notes/1.24/analysis-java.md]: Run and shown.  I've elaborated the change notes for the new queries to say this.

**Comment by felicitymay** [comment]
Does anyone else need to review this, or can I just merge it?

**Comment by aschackmull** [comment]
Feel free to merge.

**Pull request merged**